vip-helpers/vip-utils.php

```diff
@@ -405,17 +413,21 @@ function vip_redirects( $vip_redirects_array = array(), $case_insensitive = fals
  * @param bool $append_old_uri Optional. If true, the full path past the match will be added to the new URL. Defaults to false.
  */
 function vip_substr_redirects( $vip_redirects_array = array(), $append_old_uri = false ) {
-	if ( empty( $vip_redirects_array ) )
+	if ( empty( $vip_redirects_array ) ) {
 		return;
+	}
 
+	$request_uri = $_SERVER['REQUEST_URI'] ) ?? '';
```

Comment:
```suggestion
	$request_uri = $_SERVER['REQUEST_URI'] ?? '';
```
I left a closing parenthesis 🤷